sources/packages/backend/libs/sims-db/src/entities/student-assessment.model.ts

```diff
@@ -207,6 +208,19 @@ export class StudentAssessment extends RecordDataModel {
     },
   )
   disbursementSchedules?: DisbursementSchedule[];
+  /**
+   * When the reassessment happen due to an application offering change request (by institution),
+   * this will provide to the workflow the data that need be changed.
+   */
+  @ManyToOne(() => ApplicationOfferingChangeRequest, {
```

Comment:
Is it not one-to-one?

Correct me if wrong.

Reply:
Yes you are right, once the application request is approved by ministry it will be updated. It should be OneToOne relation.

Reply:
I believe that we have not been enforcing the one-to-one for similar ones in the same entity.
For instance, to be a real one-to-one on DB should we create a unique constraint on the foreign key column, right?
We can talk about having this as one-to-one on Typeorm. Is a suggestion about having the DB structure changed or only the decorator for Typeorm?

Reply:
yes, we are not using OneToOne in similar scenarios

Reply:
@ann-aot @andrewsignori-aot when I entered the comment it was keeping in mind the typeorm decorator. What you mentioned makes sense to have the enforcement in db as well. 

I saw application exception as an instance following one to one decorator. Let me know if I am missing anything.

Either way @ann-aot this comment is not a blocker as we do not have an agreement currently on this.

Reply:
@dheepak-aot @guru-aot thoughts?